marimbabot_speech/package.xml

```diff
@@ -3,20 +3,30 @@
   <name>marimbabot_speech</name>
   <version>0.0.0</version>
   <description>
-  The marimbabot_speech package is designed produce audio from a given text string, acting as a text to speech synthesis.
+  The marimbabot_speech package is designed to recognize and extract the command from the human voice, and also speech synthesis.
+    It consists of the following nodes:
+    - audio_capture: capture audio from microphone (from ros audio_common package)
+    - speech_tmp_node: convert audio to temporary file
+    - speech_stt_node: recognize speech from audio file
+    - speech_ttc_node: convert text to command
+    - speech_tts_node: convert text to speech
```

Comment:
No tts anymore

Reply:
for TTS, all the codes are from `speech_systhesis.py`, I just renamed it. Is that acceptable?

Reply:
@Flova  If you are ok with the rename, then I will merge it.